projects/aws/upgrader/LICENSE

```diff
@@ -0,0 +1,175 @@
+
```

Comment:
Can you also generate and check in the ATTRIBUTION.txt file for this project, now that it's a Go project? You will also need to copy it into the image in the Dockerfile like you do the other attributions.